apps/src/music/MusicView.jsx

```diff
@@ -326,6 +346,27 @@ class UnconnectedMusicView extends React.Component {
     // This is a way to tell React to re-render the scene, notably
     // the timeline.
     this.setState({updateNumber: this.state.updateNumber + 1});
+
+    this.saveCode();
+  };
+
+  saveCode = () => {
+    const code = Blockly.blockly_.serialization.workspaces.save(this.workspace);
+    const codeJson = JSON.stringify(code);
+    localStorage.setItem('musicLabSavedCode', codeJson);
```

Comment:
Also, maybe a question for @moneppo, but do we need to save to a different location per user? And/or do we need to clear local storage on sign in/sign out (if we don't already)?